content/en/docs/faq.md

```diff
@@ -75,3 +75,34 @@ Not natively supported but a user can configure `revocationValidations` to `skip
 Also, by validating artifact signature first we will still fail the validation if the signing identity is not trusted.
+
+## [Notary Project Terms](https://notaryproject.dev/docs/glossary/)
+
+**Q: What is Notary Project?**
+
+**A:** The Notary Project is the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all repositories within the Notary Project, including community-released tools like [notary](https://github.com/notaryproject/notary) and [notation](https://github.com/notaryproject/notation).
+
+**Q: How are Notary and Notary Project related?**
+
+**A:** Notary, a tool for artifact signing based on The Update Framework (TUF), is developed by the Notary Project community and is hosted within the Notary Project's GitHub organization. So, notary is a tool, and Notary Project is the community that developed and maintains it.
+
+**Q: What is the difference between Notary Project and TUF?**
+
+**A:** The Notary Project, as a [GitHub organization](https://github.com/notaryproject), host multiple repositories including a [TUF](https://github.com/notaryproject/tuf) repository, which is a specific repository intended for prototyping the storage of TUF metadata in OCI-compliant registries.
+
+**Q: What is Notary Project specification?**
+
+**A:** Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary Project tooling.
+
+**Q: Does Notary Project signature specification leverage TUF?**
+
+**A:** 
+The Notary Project [signature specification](https://github.com/notaryproject/specifications/blob/main/specs/signature-specification.md) does not leverage TUF. Instead, it leverages [COSE](https://github.com/notaryproject/specifications/blob/main/specs/signature-envelope-cose.md) and [JWS](https://github.com/notaryproject/specifications/blob/main/specs/) envelopes to produce and handle portable signatures. 
+
+**Q: What is the difference between Notary and Notation?**
+
+**A:** Notary and Notation are both solutions for signing artifacts. The main difference between these two tools is that notary is based on [The Update Framework](https://theupdateframework.com) (TUF) specification while Notation is non-TUF-based. While notary has server and client components, Notation has a CLI and libraries. An example of a notary implementation is Docker Content Trust (DCT). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
```

Comment:
```suggestion
**A:** [Notary](https://github.com/notaryproject/notary) and [Notation](https://github.com/notaryproject/notation) are both solutions for signing artifacts. The main difference between these two tools is that notary is based on [The Update Framework](https://theupdateframework.com) (TUF) specification while Notation is non-TUF-based. While notary has server and client components, Notation has a CLI and libraries. An example of a notary implementation is Docker Content Trust (DCT). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
```

Reply:
@Roseline-Bassey For this paragraph, you can try to combine the suggested changes from @yizha1 and me, then polish it with your own words.

Reply:
Sure, thank you.